bigbone/build.gradle

```diff
@@ -1,6 +1,11 @@
 plugins {
     id 'bigbone.library-conventions'
     id 'bigbone.integrationtest-conventions'
+    id "org.jlleitschuh.gradle.ktlint" version "11.6.1"
```

Comment:
This should go into the `buildSrc/src/main/groovy/bigbone.kotlin-conventions.gradle` file. Also, the version should be placed in the Gradle version catalog instead (`gradle/libs.versions.toml`).